extensions/qute/runtime/pom.xml

```diff
@@ -12,7 +12,7 @@
 
     <artifactId>quarkus-qute</artifactId>
     <name>Quarkus - Qute - Runtime</name>
-    <description>Qute is a templating engine designed specifically to meet the Quarkus needs</description>
+    <description>Offers templating support for web, email etc in a build time, type-safe way</description>
```

Comment:
```suggestion
    <description>Offers templating support for web, email, etc in a build time, type-safe way</description>
```

Reply:
It should be `Offer` not `Offers`, per the rules.